### PR TITLE
[Merged by Bors] - chore(MeasureTheory): replace continuity -> fun_prop

### PR DESCRIPTION
**Pull request by grunweg**
---

[Open in Gitpod](https://gitpod.io/from-referrer/)

### CODE REVIEW

File: Mathlib/MeasureTheory/Integral/PeakFunction.lean [hunk 1]
```diff
@@ -479,7 +479,7 @@ theorem tendsto_integral_comp_smul_smul_of_integrable'
       atTop (𝓝 (f 0)) := by
     apply tendsto_integral_comp_smul_smul_of_integrable hφ h'φ h If
     have A : ContinuousAt g (x₀ - 0) := by simpa using h'g
-    have B : ContinuousAt (fun x ↦ x₀ - x) 0 := Continuous.continuousAt (by continuity)
+    have B : ContinuousAt (fun x ↦ x₀ - x) 0 := by fun_prop
     exact A.comp B
```
Comment (j-loreaux) [inline Mathlib/MeasureTheory/Integral/PeakFunction.lean:483]: Does this work?
```suggestion
    exact A.comp <| by fun_prop
```

**Comment by j-loreaux** [comment]
bors d+

File: Mathlib/MeasureTheory/Integral/PeakFunction.lean (hunk 1 above)
Comment (grunweg) [reply Mathlib/MeasureTheory/Integral/PeakFunction.lean]: It does; good spot!

**Comment by grunweg** [comment]
Thanks for the review.
bors r+

**Pull request closed without merging**